frontend/src/components/labwork/step/LabworkStepOverview.tsx

```diff
@@ -126,7 +134,7 @@ const LabworkStepOverview = ({step, refreshing, setIsSorted, stepSamples, sample
                 filters={filters}
                 setFilter={setFilter}
                 setFilterOptions={setFilterOptions}
-                selection={selection}
+                selection={loading ? noopSelection : selection}
```

Comment:
I don't think the noop can bridge the gap it is on the same flag as the refreshing. If the refresh does not block the selection i doubt this will do anything for us.

Reply:
I'll try to trigger the noop with my spasms ...

Reply:
so... how did the spamming go?